core/controllers/access_validators.py

```diff
@@ -547,6 +547,34 @@ def get(self, _: str) -> None:
         pass
 
 
+class ExplorationEditorAccessValidationHandlerPage(
+    base.BaseHandler[Dict[str, str], Dict[str, str]]
+    ):
+    """The editor page for a single exploration."""
+
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                   }]
```

Comment:
Deindent this line so it matches the 'validators' indent level.

Reply:
Done